processor/src/stack/mod.rs

```diff
@@ -140,14 +142,15 @@ impl Stack {
         result
     }
 
-    /// Returns [StackOutputs] consisting of all values on the stack and all addresses in the
-    /// overflow table that are required to rebuild the rows in the overflow table.
-    pub fn build_stack_outputs(&self) -> StackOutputs {
+    /// Returns [StackOutputs] consisting of all values on the stack.
```

Comment:
(About `Stack::new()` which I can't comment directly on)

We should remove the logic in `Stack::new()` about the case `if init_values.len() > STACK_TOP_SIZE { ... }`, which can no longer happen